core/vm/actor/builtin/states/miner/miner_actor_state.cpp

```diff
@@ -454,15 +454,11 @@ namespace fc::vm::actor::builtin::states {
         .total_faulty_power = total_faulty_power};
   }
 
-  outcome::result<MinerActorStatePtr> makeEmptyMinerState(
-      const Runtime &runtime) {
-    const auto version = runtime.getActorVersion();
-    const auto ipld = runtime.getIpfsDatastore();
-
-    MinerActorStatePtr state{version};
+  outcome::result<MinerActorStatePtr> makeEmptyMinerState(const IpldPtr &ipld) {
```

Comment:
I don't see any places in your code where you need this function. It is always used by context that has "runtime" to provide it.
Why have you decided to change it?
Moreover, we could have an error if someone provides ipld without correct actor version.